### PR TITLE
Api registration

### PR DESCRIPTION
**Pull request by llaveirge**
ERRORS:
image
image

SUCCESSFUL POST & DATABASE ENTRIES:
image
image

### CODE REVIEW

**Comment by llaveirge** [comment]
I'm not sure how express-error-handling ended up here?

File: api-registration/server/index.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      params.push(hashedPassword);
+      db.query(sql, params)
+        .then(result => {
+          const [newUser] = result.rows;
+          res.status(201).json(newUser);
+        })
+        .catch(err => next(err));
+    })
+    .catch(err => {
+      console.error(err);
```
Comment (scbowler) [inline api-registration/server/index.js]: Here we would still want to call `next(err)`, the error middleware will handle console logging the error

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, 1 small update needed. Let me know if you have any questions.

File: api-registration/server/index.js (hunk 1 above)
Comment (llaveirge) [reply api-registration/server/index.js]: Ah! Yes, muscle memory kicked in there. I will get that fixed.

**Comment by llaveirge** [comment]
Still works! 

2 Registrations and errors:
image
image

Database:
image

**Review by scbowler — APPROVED** [review]
Looks good! 👍

**Pull request merged**